conjur-cli.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |gem|
   # Filter out development only executables
   gem.executables -= %w{parse-changelog.sh}
 
-  gem.add_dependency 'activesupport', '>= 4.2'
-  gem.add_dependency 'conjur-api', '~> 5.3'
+  gem.add_dependency 'activesupport', '>= 4.2', '<=6.1.4.1'
```

Comment:
Why is there so hard upper limit `<=6.1.4.1`?
It worth to leave comment with the reason.

Reply:
Thanks, changed to `'~> 6.0'`